src/backend/distributed/commands/table.c

```diff
@@ -524,12 +524,14 @@ IsAlterTableRenameStmt(RenameStmt *renameStmt)
 {
 	bool isAlterTableRenameStmt = false;
 
-	if (renameStmt->renameType == OBJECT_TABLE)
+	if (renameStmt->renameType == OBJECT_TABLE ||
+		renameStmt->renameType == OBJECT_FOREIGN_TABLE)
 	{
 		isAlterTableRenameStmt = true;
 	}
 	else if (renameStmt->renameType == OBJECT_COLUMN &&
-			 renameStmt->relationType == OBJECT_TABLE)
+		(renameStmt->relationType == OBJECT_TABLE ||
+			renameStmt->relationType == OBJECT_FOREIGN_TABLE))
```

Comment:
When you move this repo to a branch of citus, check-style test will fail. You need to run `make reindent` to reformat the changes. To be able to run that, you need to install https://github.com/citusdata/tools.